.travis.yml

```diff
@@ -25,16 +25,23 @@ jobs:
         env: TAG=$TRAVIS_COMMIT_MESSAGE
         if: branch = develop
         script:
+          - nmp i heroku-cli
+          - heroku login <<< $"$HEROKU_EMAIL\n$HEROKU_PASSWORD\n"
+          - heroku container:login
           - cd backend/nokia-innovative-project
           - mvn package -DskipTests=true
```

Comment:
package without test!

Reply:
This stage won't start if tests fail in the first stage, am I right?

Reply:
Yes, you are right. 
That's how it should work